public/style/components/pasteHeader.css

```diff
@@ -1,3 +1,7 @@
+input[type="text"] {
```

Comment:
this only affects text inputs. make it affect `*:focus` (everything in focus) so it doesnt accidentally add outlines to other elements in other browsers.

also should follow the repo style and have the brace on the new line.

Reply:
Fixed :+1: